---
bool value
mavros_msgs/Waypoint[] waypoints
